l10n_ch_hr_payroll/data/hr.salary.rule.xml

```diff
@@ -4,386 +4,396 @@ File: hr.salary.rule.xml
 Module: l10n_ch_hr_payroll
 
 Created by sge@open-net.ch
-
-Copyright (c) 2015 Open Net Sàrl
+Translated and changed source terms to English by trevi@twanda.com
+Copyright (c) 2015 Open Net Sàrl and 2016 twanda AG
 -->
 <openerp>
     <data noupdate="1">
-        <record id="AM" model="hr.salary.rule">
+        <record id="MI_GE" model="hr.salary.rule">
             <field name="amount_select">fix</field>
             <field name="appears_on_payslip">False</field>
-            <field name="code">AM</field>
+            <field name="code">MI_GE</field>
             <field name="condition_select">none</field>
             <field name="condition_range_max">999999</field>
-            <field name="name">Assurance Maternité GE</field>
+            <field name="name">Maternity Insurance GE</field>
+            <field name="note">Maternity Insurance for employees employed in the State of Geneva</field>
             <field name="quantity">1</field>
             <field name="sequence">110</field>
             <field name="category_id" ref="hr_payroll.DED"/>
         </record>
-        <record id="AM_E" model="hr.salary.rule">
+        <record id="MI_E" model="hr.salary.rule">
```

Comment:
you replaced AM with MI_GE, so it would make sense to use MI_GE_E here for consistency